source/appModuleHandler.py

```diff
@@ -563,7 +563,10 @@ def _get_productVersion(self):
 		return self.productVersion
 
 	def __repr__(self):
-		return "<%r (appName %r, process ID %s) at address %x>"%(self.appModuleName,self.appName,self.processID,id(self))

```

Comment:
is removing the address information intended?

Reply:
Yes. I've never seen a case where having the address in repr made any sense. I can easily restore it though.

Reply:
makes sense to me